frontend/packages/kubevirt-plugin/src/selectors/event/filters.ts

```diff
@@ -1,32 +1,36 @@
 import { getName, getNamespace } from '@console/shared';
 import { PodModel } from '@console/internal/models';
 import { EventInvolvedObject } from '@console/internal/module/k8s';
-import { VMIKind, VMKind } from '../../types/vm';
 import {
   VirtualMachineInstanceMigrationModel,
   VirtualMachineInstanceModel,
   VirtualMachineModel,
 } from '../../models';
 import { VIRT_LAUNCHER_POD_PREFIX } from '../../constants/vm';
+import { VMLikeEntityKind } from '../../types';
 
 type EventFilterFunction = (src: EventInvolvedObject) => boolean;
 
-const vmEventFilter = (vm: VMKind): EventFilterFunction => ({ kind, namespace, name }) =>
+const vmEventFilter = (vm: VMLikeEntityKind): EventFilterFunction => ({ kind, namespace, name }) =>
   kind === VirtualMachineModel.kind && name === getName(vm) && namespace === getNamespace(vm);
 
-const vmiEventFilter = (vm: VMKind | VMIKind): EventFilterFunction => ({ kind, namespace, name }) =>
+const vmiEventFilter = (vm: VMLikeEntityKind): EventFilterFunction => ({ kind, namespace, name }) =>
```

Comment:
lets make another type for `VMKind | VMIKind` and use these where needed

you can even do 
```tsx
export type VMILikeEntityKind = VMKind | VMIKind;
export type VMLikeEntityKind = VMILikeEntityKind | TemplateKind;
```

Reply:
there are places where we dont expect VMI. What about?

```ts
export type VMILikeEntityKind = VMKind | VMIKind;
export type VMLikeEntityKind = VMKind | TemplateKind;
export type VMLikeGenericEntityKind = VMLikeEntityKind | VMILikeEntityKind
```

still not sure about the naming
thoughts?

Reply:
edited